migrate/20230727_soft_delete_models.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:deleted_record) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :deleted_at, :timestamptz, null: false, default: Sequel::CURRENT_TIMESTAMP
+      column :model_name, :text, null: false
```

Comment:
One thing do is embed this in the model_values, either, or in addition, so it's self-describing. There are some pros and cons.